src/libraries/System.Net.Mail/tests/Unit/MailAddressTests/MailAddressDisplayNameTest.cs

```diff
@@ -21,6 +22,15 @@ public void MailAddress_WithUnicodeDisplayAndMailAddress_ToStringShouldReturnDis
             Assert.Equal(string.Format("\"{0}\" <{1}>", DisplayNameWithUnicode, Address), _mailAddress.ToString());
         }
 
+        [Fact]
+        public void MailAddress_WithDoubleQuotesDisplayAndMailAddress_ToStringShouldReturnDisplayNameEscapeSequenceAndAddressInAngleBrackets()
+        {
+            MailAddress _mailAddress = new MailAddress(Address, DisplayNameWithDoubleQuotes);
```

Comment:
Nit remove leading '_' (even though the previous test has it, it should be fixed later as well -- in a separate PR ideally).
@ManickaP has even better idea to simplify tests in this file, again that should be separate issue / PR.

Reply:
New issue: https://github.com/dotnet/runtime/issues/39625